### PR TITLE
Adds Alpine.js documentation search extension and removes second extension from PR

### PR DESCRIPTION
**Pull request by aurawindsurfing**
Adds Alpine.js documentation search extension

## Description

Adds search extension for https://alpinejs.dev/ documentation search

## Screencast

Kapture 2022-07-04 at 21 36 22

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/alpinejs-doc-search/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "alpinejs",
+  "title": "Alpinejs",
```
Comment (pernielsentikaer) [inline extensions/alpinejs-doc-search/package.json:4]:
```suggestion
  "title": "Alpine.js",
```

File: extensions/alpinejs-doc-search/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "description": "Documentation Search",
```
Comment (pernielsentikaer) [inline extensions/alpinejs-doc-search/package.json:5]:
```suggestion
  "description": "Search Alpine.js documentation",
```

File: extensions/alpinejs-doc-search/package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  "icon": "command-icon2.png",
+  "author": "aurawindsurfing",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "alpinejs",
+      "title": "Search Alpine.js Documenantion",
```
Comment (pernielsentikaer) [inline extensions/alpinejs-doc-search/package.json:12]:
```suggestion
      "title": "Search Documenantion",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @aurawindsurfing 👋 

Thanks for your contribution 💪 

I tested your extension and it works as expected, I have some suggestions and feedback for you 🙂

- Is it possible to have some initial items on the list so it's not all empty?

- You could consider a [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) to make the UI even better (with the logo in 64x64)

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by zepfietje** [comment]
@aurawindsurfing, should this extension return the same results as searching on the docs site itself? Seems like some results are missing, like `x-modelable`.

**Comment by aurawindsurfing** [comment]
@zepfietje good catch, just commited the fix as a separate PR https://github.com/raycast/extensions/pull/2238

**Comment by zepfietje** [comment]
Wow, quick fix, thanks @aurawindsurfing!

**Pull request merged**